src/truncated/exponential.jl

```diff
@@ -0,0 +1,19 @@
+#####
+##### Truncated exponential distribition
+#####
+
+function mean(d::Truncated{<:Exponential,Continuous})
+    θ = d.untruncated.θ
+    l, r = extrema(d)           # l is always finite
+    if isfinite(r)
+        if l == r
+            r
```

Comment:
with 2 embedded if else, maybe return statements would be better

Reply:
I prefer the lispy style, and would prefer not to change this unless this is the only thing that prevents this PR from being merged.